helm/provisioner/README.md

```diff
@@ -32,5 +32,114 @@ coder:
       value: "0.0.0.0:2112"
   replicaCount: 10
 provisionerDaemon:
-  pskSecretName: "coder-provisioner-psk"
+  keySecretName: "coder-provisionerd-key"
+  keySecretKey: "provisionerd-key"
 ```
+
+## Specific Examples
+
+Below are some common specific use-cases when deploying a Coder provisioner.
+
+### Set Labels and Annotations
+
+If you need to set deployment- or pod-level labels and annotations, set `coder.{annotations,labels}` or `coder.{podAnnotations,podLabels}`.
+
+Example:
+```yaml
+coder:
+  # ...
+  annotations:
+    com.coder/annotation/foo: bar
+    com.coder/annotation/baz: qux
+  labels:
+    com.coder/label/foo: bar
+    com.coder/label/baz: qux
+  podAnnotations:
+    com.coder/podAnnotation/foo: bar
+    com.coder/podAnnotation/baz: qux
+  podLabels:
+    com.coder/podLabel/foo: bar
+    com.coder/podLabel/baz: qux
+```
+
+### Additional Templates
+
+You can include extra Kubernetes manifests in `extraTemplates`.
+
+The below example will also create a `ConfigMap` along with the Helm release:
+
+```yaml
+coder:
+  # ...
+provisionerDaemon:
+  # ...
+extraTemplates:
+  - |
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: some-config
+      namespace: {{ .Release.Namespace }}
+    data:
+      key: some-value
+```
+
+### Disable Service Account Creation
+
+### Deploying multiple provisioners in the same namespace
+
+To deploy multiple provisioners in the same namespace, set the following values explicitly to avoid conflicts:
+- `nameOverride`: controls the name of the provisioner deployment
+- `serviceAccount.name`: controls the name of the service account.
+
+Note that `nameOverride` does not apply to `extraTemplates`, as illustrated below:
+
+```yaml
+coder:
+  # ...
+  serviceAccount:
+    name: other-coder-provisioner
+provisionerDaemon:
+  # ...
+nameOverride: "other-coder-provisioner"
+extraTemplates:
+	- |
+		apiVersion: v1
+		kind: ConfigMap
+		metadata:
+		  name: some-other-config
+		  namespace: {{ .Release.Namespace }}
+		data:
+		  key: some-other-value
```

Comment:
self-review: this isn't strictly necessary, but I figured it could be a potential point of confusion, so elected to clarify it here.